community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/monitoring/VisibleMigrationProgressMonitor.java

```diff
@@ -67,7 +68,7 @@ public void completed()
         public void progress( long add )
         {
             current += add;
-            int percent = max == 0 ? 100 : (int) (current*100/max);
+            int percent = max == 0 ? HUNDRED : Math.min( HUNDRED, (int) ((current * HUNDRED) / max) );
```

Comment:
This `min` here shouldn't be needed if all progress calculation works as expected, right? Having this `min` will potentially mask other issues in the future.